src/OpenTelemetry.Contrib.Extensions.AWSXRay/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
   supporting ResourceBuilderExtensions extension, and migrate all detectors to
   implement OpenTelemetry.Resources.IResourceDetector
   ([#875](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/875))
+* Drop support for `AWSLambdaResourceDetector`.
+  Resources are detected by `OpenTelemetry.Instrumentation.AWSLambda` package
```

Comment:
Resources are detected by the instrumentation package? Did you mean the new ResourceDetectors pacakge instead?

Reply:
Yes, instrumentation package contains AWSLambda detector. It is incjected when AWSLamdaInstrumentation is used.